DSCResources/xSmaVariable/xSmaVariable.schema.mof

```diff
@@ -0,0 +1,11 @@
+
+[ClassVersion("1.0.0.0"), FriendlyName("SmaVariable")]
+class xSmaVariable : OMI_BaseResource
+{
+    [Key] String Name;
+    [Required] String value;
+    [Write] String Description;
+    [Read] Boolean Set;
+    [Required] String WebServiceEndpoint;

```

Comment:
On different endpoints you can assign a variable with the same name?  If so, the WebServiceEndpoint should be part of the key.

Also, descriptions would be useful.